### ISSUE TITLE
stop closing my issues!!111!!!111

### ISSUE THREAD

**Issue by Si1kn** [Annunciate/snip-client#7]
pls listen i am a 12 yr old  i need te recxpt thanks ultragamer2919

**Comment by Premiering** [comment]
hey can you give me the source client of the code src to make the client better plesae!!!!1111!!!!

**Comment by Si1kn** [comment]
yes!!!11111!!!! also give me rank and fornite vbuc thanks!!1111111 also i like vbuc and fornite if you cant tell so give me all of yur skins so i look cool!51111111
